tests/integration/raw_studies_blueprint/test_fetch_raw_data.py

```diff
@@ -174,33 +155,63 @@ def test_get_study(
 
         # If we ask for a matrix, we should have a JSON content if formatted is True
         rel_path = "/input/links/de/fr"
```

Comment:
We should probably test it with an output matrix too, since they are handled by different code paths

Reply:
I've duplicated the input tests to do the output matrix ones.